docs/ossm/ossm2-migration/cluster-wide/README.md

```diff
@@ -0,0 +1,133 @@
+# OpenShift Service Mesh 2.6 Cluster wide --> 3 Migration guide
+This guide is for users who are currently running `ClusterWide` OpenShift Service Mesh 2.6 migrating to OpenShift Service Mesh 3.0. You should first read [this document comparing OpenShift Service Mesh 2 vs. OpenShift Service Mesh 3](../../ossm2-vs-ossm3.md) to familiarize yourself with the concepts between the two versions and the differences in managing the workloads and addons.
+
+## Migrating OpenShift Service Mesh 2.6 Cluster wide to OpenShift Service Mesh 3
+
+### Prerequisites
+- you have completed all the steps from the [pre-migration checklist](../README.md#pre-migration-checklist)
```

Comment:
Maybe flip these two steps - read OSSM 2 vs 3, then run the premigration check list. They will have more context to start this way.